minicluster/src/main/java/org/apache/accumulo/minicluster/impl/MiniAccumuloConfigImpl.java

```diff
@@ -313,6 +321,19 @@ public MiniAccumuloConfigImpl setZooKeeperStartupTime(long zooKeeperStartupTime)
   }
 
   /**
+   * Configure an existing ZooKeeper instance to use. Calling this method is optional. If not set, a new ZooKeeper instance is created.
+   *
+   * @param existingZooKeepers
+   *          Connection string for a already-running ZooKeeper instance
+   *
+   * @since 1.7.0
+   */
+  public MiniAccumuloConfigImpl setExistingZooKeepers(String existingZooKeepers) {
+    this.existingZooKeepers = existingZooKeepers;
```

Comment:
Having the set and clear operations in the same method is ok w/ me if the method is only in impl and not in API.